<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mySliderWidget</class>
 <widget class="QWidget" name="mySliderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>869</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string>音乐视频</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>883</x>
     <y>9</y>
     <width>301</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string>音乐列表</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>635</y>
     <width>1175</width>
     <height>94</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>819</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <widget class="QPushButton" name="btnPreviousSong">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-image:  url(:/images/image/pre.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover
{
background-image:url(:/images/image-hover/pre-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:  url(:/images/image/pre.png);
background-repeat:no-repeat;
background-position:center center;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>28</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/images/622.bmp</normaloff>:/images/images/622.bmp</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNextSong">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-image:  url(:/images/image/next.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover
{
background-image:url(:/images/image-hover/next-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:  url(:/images/image/next.png);
background-repeat:no-repeat;
background-position:center center;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
     </property>
    </widget>
    <widget class="QSlider" name="sliderVolumn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSound">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>819</width>
      <height>38</height>
     </rect>
    </property>
    <widget class="QSlider" name="sliderPosition">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="LabRatio">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>进度</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabCurMedia">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>无曲目</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
